README.md

```diff
@@ -232,9 +248,24 @@ we don't need to go back and edit the values returned by the ternary.
 Once we've done all that, the code defining our return value is much easier to
 read: it no longer includes a jumble of mathematical, comparison, and ternary
 expressions. Furthermore, by using meaningful variable names, we've made it
-clear what the final message will consist of:
+clear what the final message will consist of.
+
+```js
+const name = "Your name here";
+const probabilityOfRain = 0.2;
+const temperatureInC = 26;
+
+const likelyToRain = probabilityOfRain > 0.3;
+const sunIsDangerous = temperatureInC >= 26;
+
+const rainPercentage = probabilityOfRain * 100;
+const rainAdvice = likelyToRain ? "take an umbrella" : "enjoy this rain-free day";
+const sunAdvice = sunIsDangerous ? "watch out for heatstroke" : "bask in this fine weather";
+
+`Hello, ${name}, with a rain chance of ${rainPercentage}% and a temperature of ${temperatureInC}C we recommend that you ${rainAdvice} and ${sunAdvice}.`;
+```
 
-<iframe height="400px" width="100%" src="https://repl.it/@LizBurton/BitterTeemingAbandonware?lite=true" scrolling="no" frameborder="no" allowtransparency="true" allowfullscreen="true" sandbox="allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts allow-modals"></iframe>
+<iframe height="400px" width="100%" src="https://replit.com/@lizbur10/Sandbox?lite=1&outputonly=1" scrolling="no" frameborder="no" allowtransparency="true" allowfullscreen="true" sandbox="allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts allow-modals"></iframe>
```

Comment:
Maybe just add an instruction before the repl - "be sure to verify that our refactored code still works" or something